proto/message_api/v1/authn.proto

```diff
@@ -50,17 +51,19 @@ message Token {
     bytes auth_data_bytes = 2;
     // identity key signature of AuthData bytes
     Signature auth_data_signature = 3; 
-}
 
-// AuthData carries token parameters that are authenticated
-// by the identity key signature.
-// It is embedded in the Token structure as bytes
-// so that the bytes don't need to be reconstructed
-// to verify the token signature.
-message AuthData {
-    // address of the wallet
-    string wallet_addr = 1;
-    // time when the token was generated/signed 
-    uint64 timestamp = 2; 
+    // AuthData carries token parameters that are authenticated
+    // by the identity key signature.
+    // It is embedded in the Token structure as bytes
+    // so that the bytes don't need to be reconstructed
+    // to verify the token signature.
+    message AuthData {
```

Comment:
Why do we want this inside the Token message when it isn't actually used in the Token message?

I'm fine with it being a separate message

Reply:
I was just trying to organize things more uniformly, types that are only used in a message to be inner types of the message. But I can see the argument that AuthData is used indirectly, I can move it back out.